content/en/docs/collector/howto-receiver-trace.md

```diff
@@ -1,8 +1,10 @@
-# Building a Traces Receiver
+---
+title: Building a Trace Receiver
+---
```

Comment:
@rquedas - I've added Hugo front-matter so that the page gets properly published and listed in the side-nav, etc.

All: I made "Trace" singular.